wallet/src/de/schildbach/wallet/ui/TransactionResultActivity.kt

```diff
@@ -164,6 +172,15 @@ class TransactionResultActivity : AbstractWalletActivity() {
         dashPayViewModel = ViewModelProvider(this).get(DashPayViewModel::class.java)
     }
 
+    override fun finish() {
+        val dashPayUsername = intent.extras?.getString(EXTRA_DASHPAY_USERNAME)
+        val lastLoadedUser = PlatformRepo.getInstance().lastLoadedUser
```

Comment:
In my Story for paying pending contacts (and also Contacts) from a QR code, I will need an observer.  Maybe you don't need to change this to use an observer, since I will be doing it.

Reply:
Nevermind, my liveData doesn't work for your case because my liveData is from the local database, but stranger data comes from Platform.